view/theme/frio/js/theme.js

```diff
@@ -795,7 +795,8 @@ function doActivityItemAction(ident, verb, un) {
 	$('button[id^=' + verb + '-' + ident.toString() + '] i:first-child').removeClass(thumbsClass);
 	// if verb is announce, then one rotator is added above to the shareMedia-dropdown button
 	if ($('button:not(button.dropdown-toggle) img#waitfor-' + verb + '-' + ident.toString()).length == 0) {
-		$('<img id="waitfor-' + verb + '-' + ident.toString() + '">')
+		$('<img>')
+      .attr('id', 'waitfor-' + verb + '-' + ident.toString())
 			.attr('src', 'images/rotator.gif')
 			.addClass('fa')
 			.appendTo($('button[id^=' + verb + '-' + ident.toString() + '] i:first-child')).show();
```

Comment:
Is the `show()` call necessary?
```suggestion
			.appendTo($('button[id^=' + verb + '-' + ident.toString() + '] i:first-child'))
			.show();
```

Reply:
not neccessary. removed it.